server/config.js

```diff
@@ -81,6 +87,12 @@ var conf = convict({
         doc: 'Require SSL for postgres connections.',
         default: false,
         format: Boolean
+      },
+      queryTimeout: {
+        doc: 'Milliseconds before app-level queries will bail',
+        default: 15000,
+        env: 'PGTIMEOUT',
```

Comment:
yeah, the PG* env vars are the ones natively supported by PG; it seemed weird to break compatibility just for the sake of config uniformity. I actually might just remove all the env vars. I don't like them, and ops doesn't plan to use them.